controllers/provisioning_controller.go

```diff
@@ -78,7 +82,12 @@ func (r *ProvisioningReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error
 		return ctrl.Result{}, errors.Wrap(err, "could not determine whether to run")
 	}
 	if !enabled {
-		// TODO: Set ClusterOperator status to Disabled.
+		// set ClusterOperator status to disabled=true, available=true
+		err = r.updateCOStatusDisabled()
+		if err != nil {
+			return ctrl.Result{}, err
+		}
+
 		// We're disabled; don't requeue
```

Comment:
If updateCOStatusAvailable() is going to be part of this commit, we should set the CO status to Available here when the Platform type is Baremetal.